.github/workflows/DevEnv-frontend-BuildImageAndPush.yml

```diff
@@ -7,111 +7,96 @@ env:
   IMAGE_REGISTRY_USER: ${{ secrets.C5839F_OPENSHIFT_GITHUB_ACTIONS_SA_USER }}
   IMAGE_REGISTRY_PASSWORD: ${{ secrets.C5839F_OPENSHIFT_GITHUB_ACTIONS_SA_TOKEN }}
   IMAGE_NAME: mspdirect-frontend
+  DEV_SUFFIX: -dev
   IMAGE_TAGS:
 
 on:
   # Run the workflow manually, instead of on a push.
-  # The user specifies the name of the build config to use, which should be an
-  #   API build config.
+  # The user specifies the name of the branch which is defaulted to 'main'.
   # --------------------------------------------------------------------------
   workflow_dispatch:
     inputs:
       branch:
-        description: 'Enter branch name:'
+        description: "Enter branch name:"
         required: true
         default: main
 
-
 jobs:
-  
   buildAndPush:
     runs-on: ubuntu-latest
-    
-    steps:
-    # Check out the code
-    - name: Check out code
-      id: checkout
-      uses: actions/checkout@v2
-      with:
-        ref: ${{ github.event.inputs.branch }}        
-    
 
-    # Determine the tags
-    - name: Determine image tags
-      if: env.IMAGE_TAGS == ''
-      run: |
-        echo "IMAGE_TAGS=${GITHUB_SHA::12}" | tee -a $GITHUB_ENV
+    steps:
+      # Check out the code
+      - name: Check out code
+        id: checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
 
-    # Determine the version number tag i.e. Release_1.0_<branch-name>_<build execution> 
-    - name: Determine Version number
-      # Branch name is fixed to 'main' in version identifier
-      run: |
-        echo "RELEASE_VERSION=Dev_Build_${{ github.event.inputs.branch }}_${{github.run_number}}" | tee -a $GITHUB_ENV
+      # Determine the version number tag i.e. Release_1.0_<branch-name>_<build execution>
+      - name: Determine Version number
+        # Branch name is taken from action inputs
+        run: |
+          echo "VERSION=Dev_Build_${{ github.event.inputs.branch }}_${{github.run_number}}" | tee -a $GITHUB_ENV
 
-    # Edit .env in frontend with the value
-    - name: Edit env file to add version
-      run: |
-        cd ./frontend/env_config
-        sed -i 's/<VERSION>/${{ env.RELEASE_VERSION }}/' .env
+      # Edit .env in frontend with the value
+      - name: Edit env file to add version
+        run: |
+          cd ./frontend/env_config
+          sed -i 's/<VERSION>/${{ env.VERSION }}/' .env
 
-    # Build from docker file
-    - name: Build from Dockerfile
-      id: build-docker
-      uses: redhat-actions/buildah-build@v2
-      with:
-        image: ${{ env.IMAGE_NAME }}
-        tags: ${{ env.IMAGE_TAGS }}
-        context: ./frontend
-        dockerfiles: |
-          ./frontend/Dockerfile
+      # Determine the tags
+      - name: Determine image tags
+        if: env.IMAGE_TAGS == ''
```

Comment:
we dont need this if check any more. Same for all the actions

Reply:
Removed all these unnecessary checks